patch/revert-456d8aa-to-fix-pcie_aspm_exit_link_status.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nathan Wolfe <nwolfe@arista.com>
+Date: Tue, 8 Oct 2024 11:57:26 -0700
+Subject: [PATCH] revert 456d8aa to fix pcie_aspm_exit_link_status
+
```

Comment:
Please add a summary, and also a reference to the upstream discussion.

Reply:
Please add a link to the upstream discussion in the patch itself here.